### PR TITLE
Android Test Readme

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

File: android-test/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Android Test
+============
+
+A gradle module for running Android instrumentation tests on device or emulator.
```
Comment (swankjesse) [inline android-test/README.md]: on _a_ device or emulator?

**Comment by swankjesse** [comment]
Handy!

**Comment by swankjesse** [comment]
We should get the website generator script to pull in these files. We're overdue for a mockwebserver page on our site!

**Comment by yschimke** [comment]
Example output

```
01-01 13:19:26.489 13169 13190 I TestRunner: started: testCertificatePinningSuccess(okhttp.android.test.OkHttpTest)
01-01 13:19:26.491 13169 13190 I TaskRunner: Q10000 scheduled after   0 µs: MockWebServer 52553
01-01 13:19:26.492 13169 13196 I TaskRunner: Q10000 starting              : MockWebServer 52553
01-01 13:19:26.509 13169 13196 I TaskRunner: Q10001 scheduled after   0 µs: MockWebServer /127.0.0.1:39358
01-01 13:19:26.510 13169 13197 I TaskRunner: Q10001 starting              : MockWebServer /127.0.0.1:39358
01-01 13:19:26.536 13169 13197 I Http2   : >> 0x00000000     0 SETTINGS
01-01 13:19:26.536 13169 13190 I Http2   : >> CONNECTION 505249202a20485454502f322e300d0a0d0a534d0d0a0d0a
01-01 13:19:26.536 13169 13190 I Http2   : >> 0x00000000     6 SETTINGS
01-01 13:19:26.537 13169 13197 I TaskRunner: Q10001 finished run in  27 ms: MockWebServer /127.0.0.1:39358
01-01 13:19:26.537 13169 13190 I Http2   : >> 0x00000000     4 WINDOW_UPDATE
01-01 13:19:26.538 13169 13199 I Http2   : << CONNECTION 505249202a20485454502f322e300d0a0d0a534d0d0a0d0a
01-01 13:19:26.538 13169 13190 I TaskRunner: Q10001 scheduled after   0 µs: OkHttp ConnectionPool
01-01 13:19:26.538 13169 13199 I Http2   : << 0x00000000     6 SETTINGS
01-01 13:19:26.540 13169 13199 I TaskRunner: Q10002 scheduled after   0 µs: MockWebServer localhost applyAndAckSettings
01-01 13:19:26.541 13169 13197 I TaskRunner: Q10002 starting              : MockWebServer localhost applyAndAckSettings
01-01 13:19:26.542 13169 13197 I TaskRunner: Q10004 scheduled after   0 µs: MockWebServer localhost onSettings
01-01 13:19:26.544 13169 13197 I Http2   : >> 0x00000000     0 SETTINGS      ACK
01-01 13:19:26.544 13169 13200 I Http2   : << 0x00000000     0 SETTINGS
01-01 13:19:26.544 13169 13201 I TaskRunner: Q10001 starting              : OkHttp ConnectionPool
01-01 13:19:26.545 13169 13200 I TaskRunner: Q10004 scheduled after   0 µs: OkHttp localhost applyAndAckSettings
01-01 13:19:26.545 13169 13199 I Http2   : << 0x00000000     4 WINDOW_UPDATE
01-01 13:19:26.545 13169 13197 I TaskRunner: Q10002 finished run in   3 ms: MockWebServer localhost applyAndAckSettings
01-01 13:19:26.545 13169 13197 I TaskRunner: Q10004 starting              : MockWebServer localhost onSettings
01-01 13:19:26.545 13169 13197 I TaskRunner: Q10004 finished run in  70 µs: MockWebServer localhost onSettings
01-01 13:19:26.545 13169 13201 I TaskRunner: Q10001 run again after 300 s : OkHttp ConnectionPool
01-01 13:19:26.546 13169 13201 I TaskRunner: Q10001 finished run in   2 ms: OkHttp ConnectionPool
01-01 13:19:26.546 13169 13200 I Http2   : << 0x00000000     0 SETTINGS      ACK
01-01 13:19:26.546 13169 13201 I TaskRunner: Q10004 starting              : OkHttp localhost applyAndAckSettings
01-01 13:19:26.548 13169 13190 I Http2   : >> 0x00000003    40 HEADERS       END_STREAM|END_HEADERS
01-01 13:19:26.548 13169 13201 I TaskRunner: Q10006 scheduled after   0 µs: OkHttp localhost onSettings
01-01 13:19:26.548 13169 13201 I Http2   : >> 0x00000000     0 SETTINGS      ACK
01-01 13:19:26.551 13169 13204 I TaskRunner: Q10006 starting              : OkHttp localhost onSettings
01-01 13:19:26.551 13169 13201 I TaskRunner: Q10004 finished run in   5 ms: OkHttp localhost applyAndAckSettings
01-01 13:19:26.552 13169 13204 I TaskRunner: Q10006 finished run in 426 µs: OkHttp localhost onSettings
01-01 13:19:26.593 13169 13199 I Http2   : << 0x00000003    40 HEADERS       END_STREAM|END_HEADERS
01-01 13:19:26.595 13169 13199 I TaskRunner: Q10005 scheduled after   0 µs: MockWebServer localhost[3] onStream
01-01 13:19:26.595 13169 13199 I Http2   : << 0x00000000     0 SETTINGS      ACK
01-01 13:19:26.595 13169 13202 I TaskRunner: Q10005 starting              : MockWebServer localhost[3] onStream
01-01 13:19:26.599 13169 13202 I Http2   : >> 0x00000000     0 SETTINGS
01-01 13:19:26.599 13169 13200 I Http2   : << 0x00000000     0 SETTINGS
01-01 13:19:26.599 13169 13200 I TaskRunner: Q10004 scheduled after   0 µs: OkHttp localhost applyAndAckSettings
01-01 13:19:26.600 13169 13201 I TaskRunner: Q10004 starting              : OkHttp localhost applyAndAckSettings
01-01 13:19:26.600 13169 13201 I TaskRunner: Q10006 scheduled after   0 µs: OkHttp localhost onSettings
01-01 13:19:26.600 13169 13201 I Http2   : >> 0x00000000     0 SETTINGS      ACK
01-01 13:19:26.600 13169 13204 I TaskRunner: Q10006 starting              : OkHttp localhost onSettings
01-01 13:19:26.600 13169 13204 I TaskRunner: Q10006 finished run in 189 µs: OkHttp localhost onSettings
01-01 13:19:26.600 13169 13201 I TaskRunner: Q10004 finished run in 689 µs: OkHttp localhost applyAndAckSettings
01-01 13:19:26.601 13169 13199 I Http2   : << 0x00000000     0 SETTINGS      ACK
01-01 13:19:26.601 13169 13202 I Http2   : >> 0x00000003     7 HEADERS       END_HEADERS
01-01 13:19:26.602 13169 13202 I Http2   : >> 0x00000003     3 DATA
01-01 13:19:26.603 13169 13200 I Http2   : << 0x00000003     7 HEADERS       END_HEADERS
01-01 13:19:26.604 13169 13200 I Http2   : << 0x00000003     3 DATA
01-01 13:19:26.604 13169 13202 I Http2   : >> 0x00000003     0 DATA          END_STREAM
01-01 13:19:26.604 13169 13202 I TaskRunner: Q10005 finished run in   9 ms: MockWebServer localhost[3] onStream
01-01 13:19:26.605 13169 13190 I Http2   : >> 0x00000003     4 RST_STREAM
01-01 13:19:26.606 13169 13199 I Http2   : << 0x00000003     4 RST_STREAM
01-01 13:19:26.606 13169 13190 I TaskRunner: Q10001 scheduled after   0 µs: OkHttp ConnectionPool
01-01 13:19:26.606 13169 13200 I Http2   : << 0x00000003     0 DATA          END_STREAM
01-01 13:19:26.606 13169 13204 I TaskRunner: Q10001 starting              : OkHttp ConnectionPool
01-01 13:19:26.606 13169 13204 I TaskRunner: Q10001 run again after 300 s : OkHttp ConnectionPool
01-01 13:19:26.606 13169 13204 I TaskRunner: Q10001 finished run in 245 µs: OkHttp ConnectionPool
01-01 13:19:26.606 13169 13200 I TaskRunner: Q10004 scheduled after   0 µs: OkHttp localhost[3] writeSynReset
01-01 13:19:26.607 13169 13206 I TaskRunner: Q10004 starting              : OkHttp localhost[3] writeSynReset
01-01 13:19:26.607 13169 13206 I Http2   : >> 0x00000003     4 RST_STREAM
01-01 13:19:26.608 13169 13206 I TaskRunner: Q10004 finished run in 711 µs: OkHttp localhost[3] writeSynReset
01-01 13:19:26.608 13169 13199 I Http2   : << 0x00000003     4 RST_STREAM
01-01 13:19:26.610 13169 13190 I TaskRunner: Q10000 scheduled after   0 µs: OkHttp awaitIdle
01-01 13:19:26.612 13169 13196 I Http2   : >> 0x00000000     8 GOAWAY
01-01 13:19:26.612 13169 13200 I Http2   : << 0x00000000     8 GOAWAY
01-01 13:19:26.613 13169 13196 I TaskRunner: Q10000 finished run in 122 ms: MockWebServer 52553
01-01 13:19:26.614 13169 13196 I TaskRunner: Q10000 starting              : OkHttp awaitIdle
01-01 13:19:26.614 13169 13196 I TaskRunner: Q10000 finished run in 204 µs: OkHttp awaitIdle
01-01 13:19:26.615 13169 13190 I TestRunner: finished: testCertificatePinningSuccess(okhttp.android.test.OkHttpTest)
```

**Pull request merged**